jobs/build/aws-ami/Jenkinsfile

```diff
@@ -25,7 +25,7 @@ properties(
                                                     openshift-enterprise      Openshift Enterpriseonline <br>
                                     ''',
                                     name: 'DEPLOYMENT_TYPE'],
-                                 [$class: 'hudson.model.StringParameterDefinition', defaultValue: 'opstest', description: 'Clusterid to set for ami build. Also used for VPC and other settings', name: 'CLUSTERID'],
+                                 [$class: 'hudson.model.StringParameterDefinition', defaultValue: 'default', description: 'Clusterid to set for ami build. Also used for VPC and other settings', name: 'CLUSTERID'],
```

Comment:
I changed this to use the `default` cluster and vpc (which the account has).
Is that a reasonable value for this variable?

Reply:
I think its safe to change this to something else but what that something else is I'm not sure.  There are discussions ongoing about where this work will land.  When it lands and where it lands (which AWS account) will then help us better determine what this value should be.  

I think the push will be to get CICD team their own AWS account so that they can build a space for jobs like these that require some known set of environment and infrastructure settings.  Until then feel free to change this to whatever makes it work.